code/objects/items/equipable/belt.dm

```diff
@@ -1,5 +1,7 @@
 /obj/item/equipable/belt
-	icon = 'icons/items/belt.dmi'
+	layer = BELT_LAYER
```

Comment:
don't set `layer` on an object that exists in the world to `FLOAT_LAYER`, add a variable like `worn_layer` or something.